src/contentProvider.ts

```diff
@@ -8,6 +8,12 @@ import { MavenProject } from "./explorer/model/MavenProject";
 import { getDependencyTree } from "./handlers/showDependenciesHandler";
 import { Utils } from "./utils/Utils";
 
+/**
+ * URI patterns.
+ * vscode-maven://dependencies/<pom-path>/Dependencies?<pom-path>
+ * vscode-maven://effective-pom/<pom-path>/EffectivePOM.xml?<pom-path>
+ * vscode-maven:///<pom-path-in-local-maven-repository>
```

Comment:
three `/`s here, as expected?

Reply:
as expected. Here authority is empty, so `//<empty>` is authority. And `path` starts with "/".

> URI = scheme:[//authority]path[?query][#fragment]

I use `authority` to differentiate dependencies/effective-pom/common pom file.